### DISCUSSION TITLE
After restoring a dump file I can't run migrate anymore

### DISCUSSION THREAD

**Discussion by Akumzy** [prisma/prisma#13124] [Q&A]
After restoring a dump file I can't run migrate anymore as it kept on throwing this error, and I have tried resetting the DB using `npx prisma migrate reset` and that command worked
```log
Error: P3014

Prisma Migrate could not create the shadow database. Please make sure the database user has permission to create databases. Read more about the shadow database (and workarounds) at https://pris.ly/d/migrate-shadow

Original error: Error code: P1003

Database `(not available)` does not exist on the database server at `localhost:5432`.

```

It's reading the database/writing but migration is not working.
Please what am I doing wrong or have done wrong?

**Comment by nurul3101** [comment]
Hey @Akumzy 👋 

It seems that you are using Azure SQL or Heroku which doesn't allow the automatic creation of shadow databases.
Please take a look at how you could create the shadow database manually: [Shadow Database](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually)

Alternatively, if you are just prototyping you could use `npx prisma db push` to propagate your database changes. db push command doesn't use shadow database.

**Reply by Akumzy** [reply]
HI @nurul3101, No all these are happening locally, I needs up uninstalling Postgres and reinstalled it

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
